frontend/misc.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * SPDX-FileCopyrightText: 2022 Zeal 8-bit Computer <contact@zeal8bit.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+$(".tab").on("click", function(){
+    const index = $(this).index();
+    $(".tab").removeClass("active");
+    $(".bottompanel .panel").addClass("hidden");
+    $(".bottompanel .panel").eq(index).removeClass("hidden");
+    $(this).addClass("active");
+});
+
+$(".regaddr").click(function() {
```

Comment:
This is part of the register panel (left) so should it be moved to `panel.js`?

Reply:
Modified.